src/math/mat3.js

```diff
@@ -78,6 +60,33 @@ Object.assign(pc, (function () {
             return this;
         },
 
+        /**
+         * @function
+         * @name pc.Mat3#set
+         * @description Copies the contents of a source array[9] to a destination 3x3 matrix.
+         * @param {Array} src An array[9] to be copied.

```

Comment:
I think my preference here is to have the function take 9 number arguments than 1 array. The problem being that we already have pc.Mat4#set that takes an array. As you can see below where you have updated the creation of pc.Mat3#ZERO, you've had to allocate an array redundantly. I wonder how many people currently use pc.Mat4#set...

Reply:
It's not a problem at all to change this - my intention was to keep M3 and M4 interface consistent. So let me know if 9 args is more preferable over an array.

Reply:
As discussed, let's leave this as a single array argument for now, then.